imagecraft/subprocesses.py

```diff
@@ -39,12 +36,6 @@ def run(cmd: str, *args: Any, **kwargs: Any) -> CompletedProcess[str]:
         if key not in kwargs:
             kwargs[key] = value
 
-    msg = f"Command: {cmd} {shlex.join(str(a) for a in args)}"
-    try:
-        emit.debug(msg)
```

Comment:
Was this related to the problem where packing would only succeed in some verbosity level?